gridcapa-swe-commons/src/test/java/com/farao_community/farao/gridcapa_swe_commons/shift/CountryBalanceComputationTest.java

```diff
@@ -4,20 +4,20 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
-package com.farao_community.farao.swe.runner.app.dichotomy.shift;
+package com.farao_community.farao.gridcapa_swe_commons.shift;
 
 import com.powsybl.iidm.network.Network;
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
 
 import java.util.Map;
 
-import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
 /**
  * @author Ameni Walha {@literal <ameni.walha at rte-france.com>}
  */
-@SpringBootTest
+@SpringBootTest(classes = CountryBalanceComputation.class)
```

Comment:
You don't need @SpringBootApplication annotation here as you test class' methods and not instance's methods